scripts/ci/ci-run-stateful-tests-cluster.sh

```diff
@@ -47,7 +47,7 @@ fi
 echo "Starting stateful databend-test"
 # Create table
 # shellcheck disable=SC2044
-for i in $(find ./tests/suites/0_stateful/sql -name "*.sql" -type f); do
+for i in $(find ./tests/suites/0_stateful/01_ontime/sql -name "*.sql" -type f); do
```

Comment:
Seems this script is duplicated with `databend-test`.

For stateful tests:

Step 1:  Load DDL SQL queries to create tables.
Step 2:  Load sample data into the tables.
Step 3: use `./databend-test --run-dir 0_stateful`

Reply:
Ok.

This PR try to re-fine the stateful test directory layout, to make it more readable.
For example:
01_ontime -- this is a ontime dataset test.

I am not sure the ontime dataset willbe used by other stateful tests.

Reply:
Change it to:
```
SCRIPT_PATH="$(cd "$(dirname "$0")" >/dev/null 2>&1 && pwd)"
cd "$SCRIPT_PATH/../../tests" || exit

echo "Starting databend-test"
./databend-test --mode 'cluster' --run-dir 0_stateful
```

but got:
```
successfull loaded ontime dataset
Starting stateful databend-test
No tests were run.
make: *** [Makefile:119: stateful-cluster-test] Error 1
```